### PR TITLE
update proposal use cases and related solutions

### PR DESCRIPTION
**Pull request by sjenning**
sync for Paul and me @pmorie

### CODE REVIEW

File: docs/proposals/service-catalog.md [hunk 1]
```diff
@@ -33,27 +33,167 @@ Documentation for other releases can be found at
...
+the template, they are not tracked any further as a member of the template.  It
+is up to the users to apply labels such that all resources created by a template
+can be selected in the cluster.
+
+### Cloud Foundry Service Broker
+
+The CF Service Broker is more full featured than the previous examples, in that
+it implements a catalog of services (CCDB) provided by any number of service
+brokers and claimable by cluster users.  The cloud controller can discover
+services from service brokers that implementto toss it. the Service Broker API
```
Comment (ncdc) [inline docs/proposals/service-catalog.md:182]: "that implementto toss it" ?

File: docs/proposals/service-catalog.md [hunk 2]
```diff
@@ -33,27 +33,167 @@ Documentation for other releases can be found at
...
+using the "catalog" call.  Other calls that happen in response to a user
+claiming the service from the catalog are "provision", "bind", "unbind", and
+"deprovision".  The separation of the provisioning stage from the binding stage
+allows for asynchronous dynamic provisioning of a service for a particular user.
+
+This satisfies all the use cases except for the syncing of changed credentials.
+There is no mechanism for the Service Broker to initiate a deprovision or
+resync.
+
+## The Thing
```
Comment (ncdc) [inline docs/proposals/service-catalog.md:192]: ?

File: docs/proposals/service-catalog.md [hunk 3]
```diff
@@ -33,27 +33,167 @@ Documentation for other releases can be found at
...
+
+## The Prototype Thing
```
Comment (ncdc) [inline docs/proposals/service-catalog.md:194]: ?

**Comment by sjenning** [comment]
i'm closing this one. working in paul's branch/PR

**Pull request closed without merging**